doc/python/2D-Histogram.md

```diff
@@ -70,6 +70,33 @@ fig = go.Figure(go.Histogram2d(x=x, y=y, histnorm='probability',
     ))
 fig.show()
 ```
+### Sharing bin settings between 2D Histograms
+This example shows how to use [bingroup](https://plot.ly/python/reference/#histogram-bingroup) attribute to have a compatible bin settings for both histograms. To define `start`, `end` and `size` value of x-axis and y-axis seperatly, set [ybins](https://plot.ly/python/reference/#histogram2dcontour-ybins) and `xbins`.
+
+```python
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+fig = make_subplots(2,2)
+fig.add_trace(go.Histogram2d(
+    x = [ 1, 2, 2, 3, 4 ],
+    y = [ 1, 2, 2, 3, 4 ],
+    xbins = {'start':1, 'size':1}), 1,1)
```

Comment:
sorry one last comment: these subplots should share the same coloraxis, at the moment the colorbar has several sets of labels superimposed